setup.py

```diff
@@ -30,4 +30,21 @@
         'httmock==1.2.6',
         'mock==2.0.0',
         ],
+    extras_require = {
+        'dataframe': [
+            'geopandas==0.4.0',
```

Comment:
Do the versions need to be exact? Can we be more forgiving towards the current state of the users environment?

Reply:
They can be inexact.